src/features/people/PeoplePage/index.js

```diff
@@ -10,46 +10,61 @@ import ErrorPage from "../../../common/ErrorPage";
 import FeatureLink from "../../../common/FeatureLink";
 import { fetchPopularPeople, selectPopularPeople, selectPopularPeopleLoadingState, selectPopularPeopleErrorState, resetPopularPeople } from "../popularPeopleSlice";
 import { useQueryParameter } from "../../../useQueryParameters";
+import { search as searchParameterName } from "../../../queryParamNames";
 import { toPerson } from "../../../routes";
+import NoResultsPage from "../../../common/NoResultsPage";
 
 const PeoplePage = () => {
 
   const dispatch = useDispatch();
-  const popularPeople = useSelector(selectPopularPeople).results;
-  const popularPeopleLoading = useSelector(selectPopularPeopleLoadingState);
-  const popularPeopleError = useSelector(selectPopularPeopleErrorState);
-  const query = useQueryParameter(pageParameterName);
+  const people = useSelector(selectPopularPeople);
+  const loading = useSelector(selectPopularPeopleLoadingState);
+  const error = useSelector(selectPopularPeopleErrorState);
+  const page = useQueryParameter(pageParameterName);
+  const query = useQueryParameter(searchParameterName);
 
   useEffect(() => {
-    dispatch(fetchPopularPeople(query || 1));
+    dispatch(fetchPopularPeople({ page: page || 1, query }));
+    console.log(people);
     return (() => {
       dispatch(resetPopularPeople())
```

Comment:
Ten reset tutaj będzie powodował, że po wejściu w danego człowieka:D i potem po kliknięciu "wstecz" zamiast wrócić do wyników wyszukania będzie wyświetlało od nowa popular people. 
Z punktu widzenia UX to turbo złe.